src/service/viewport/viewport-impl.js

```diff
@@ -1169,7 +1169,8 @@ function createViewport(ampdoc) {
   let binding;
   if (
     ampdoc.isSingleDoc() &&
-    getViewportType(win, viewer) == ViewportType.NATURAL_IOS_EMBED
+    getViewportType(win, viewer) == ViewportType.NATURAL_IOS_EMBED &&
+    !IS_SXG
```

Comment:
I know I asked this before but to be completely certain: we do not believe that SXG would ever run in an iframe?

Reply:
Last I read the spec, it's a requirement that `window.parent === window` in SXG.

Reply:
👍